src/common/tracing/src/panic_hook.rs

```diff
@@ -14,23 +14,25 @@
 
 use std::panic::PanicHookInfo;
 use std::sync::atomic::Ordering;
+use std::sync::Arc;
 
 use backtrace::Backtrace;
 use backtrace::BacktraceFrame;
 use databend_common_base::runtime::LimitMemGuard;
 use databend_common_exception::USER_SET_ENABLE_BACKTRACE;
 use log::error;
 
-pub fn set_panic_hook() {
+pub fn set_panic_hook(version: String) {
```

Comment:
Why add the `version` parameter here? The crash hook can directly access `databend_common_base::version::DATABEND_COMMIT_VERSION`

Reply:
If we grep the backtrace as keyword, it can not display the version info.

It's better to set them into one row.